### DISCUSSION TITLE
How to modify the number of selected ensmble by gajoe

### DISCUSSION THREAD

**Discussion by zzzichenkeke** [biosaxs-com/atsas-community#1950] [Mixtures and flexible systems] [answered]
Hi, sorry to bother. I am confused by the process of gajoe calculation for selecting out the best fitting ensembles. I tried several times with my own pool (including 201 models), and no matter what parameters I adjusted (like Number of ensembles, Maximum number of curves per ensemble, etc.), it automatically selected out two best fitting models. I would like to get five best models for further analysis, does any parameter matter? Or it is actually related to my inputting pool, shall I enrich the size of my pool? Thanks!

**Comment by franke-biosaxs** [comment, marked as answer]
gajoe (and nnlsjoe) are to pick out a few models out of a large pool. If you want the "N best" models out of small pool, then simply use crysol on each and pick those with the best chi-square.

**Reply by zzzichenkeke** [reply]
I got it! Thanks a lot!
